packages/docs/pages/button-group.tsx

```diff
@@ -114,15 +114,27 @@ const ButtonGroupPage = () => {
       <Panel header="Do's and Don'ts" headerId="guidelines">
         <GuidelinesTable
           recommended={[
-            'Hide secondary actions or actions that doesn’t fit the container under ellipsis.',
-            'Hide destructive actions under ellipsis.',
-            'Keep all the controls in the same outlined style.',
-            'Show only actions that are available for the entity.',
+            <>
+              Hide <Code>secondary</Code> actions or <Code primary>actions</Code> that doesn’t fit the container under
```

Comment:
Playing grammar police here, shouldn't this be `that don't fit` ?

I might be wrong here, but it caught my eye

Reply:
Good catch, I think you are right, since is referring to the actions.